arch/xtensa/src/esp32s3/esp32s3_qspi.c

```diff
@@ -1529,6 +1529,56 @@ int esp32s3_qspibus_set_attr(struct qspi_dev_s *dev,
   return OK;
 }
 
+/****************************************************************************
+ * Name: esp32s3_qspibus_set_delay
+ *
+ * Description:
+ *   Set timing delay for QSPI input and output data.
+ *
+ * Input Parameters:
+ *   dev        - Device-specific state data
+ *   din_mode   - Input mode(0~3) to delay input data
+ *   din_num    - The delay number(0~3) to input data
+ *   dout_mode  - Output mode(0,1) to delay output data
+ *
+ * Returned Value:
+ *   Zero (OK) is returned on success.  Otherwise -1 (ERROR).
+ *
+ ****************************************************************************/
+
+int esp32s3_qspibus_set_delay(struct qspi_dev_s *dev,
+                                uint8_t din_mode,
+                                uint8_t din_num,
+                                uint8_t dout_mode)
```

Comment:
Please fix alignment: all uint8_t should be left aligned with struct name from previous line